src/app/(proper_react)/(redesign)/(authenticated)/admin/announcements/AnnouncementsAdmin.tsx

```diff
@@ -156,315 +159,299 @@ export const AnnouncementsAdmin = (props: Props) => {
     }
   }, [announcements, activeAnnouncementId]);
 
-  // States for each image
-  const [smallImageIsLoading, setSmallImageIsLoading] = useState(true);
-  const [bigImageIsLoading, setBigImageIsLoading] = useState(true);
-  const [smallImageUnavailable, setSmallImageUnavailable] = useState(false);
-  const [bigImageUnavailable, setBigImageUnavailable] = useState(false);
-
   const smallImagePath = `/images/announcements/${activeAnnouncement?.announcement_id.trim()}/small.svg`;
   const bigImagePath = `/images/announcements/${activeAnnouncement?.announcement_id.trim()}/big.svg`;
 
-  useEffect(() => {
-    setSmallImageIsLoading(true);
-    setBigImageIsLoading(true);
-    setSmallImageUnavailable(false);
-    setBigImageUnavailable(false);
-  }, [activeAnnouncementId]);
-
   const sortedAnnouncements = [...announcements].sort((a, b) => {
     return new Date(b.updated_at).getTime() - new Date(a.updated_at).getTime();
   });
+  const [activeTab, setActiveTab] = useState<
+    "announcements" | "strings" | "docs"
+  >("announcements");
+  const [searchQuery, setSearchQuery] = useState("");
 
   return (
     <div className={styles.container}>
-      <div className={styles.wrapper}>
-        {/* List of announcements */}
-        <div className={styles.notificationsWrapper}>
-          <h1>All Announcements</h1>
-          <ul>
-            {sortedAnnouncements.map((notification) => (
-              <li
-                key={notification.id}
-                className={
-                  activeAnnouncementId === notification.id ? styles.active : ""
-                }
-                onClick={() => handleClick(notification.id)}
-              >
-                <div>
-                  <p className={styles.title}>
+      <div className={styles.tabBar}>
```

Comment:
Again, not a blocker since it's "just" an admin page, but [react-aria implements tab behaviour](https://react-spectrum.adobe.com/react-aria/useTabList.html) - e.g. to make sure you can switch tabs using the arrow keys, and to announce them to screen readers. Just FYI for the future.